src/background/wallet.js

```diff
@@ -52,22 +53,22 @@ export async function init() {
           return;
         }
 
-        addAeppConnection(port);
+        await addAeppConnection(port);
         break;
       }
       default:
         throw new Error('Unknown connection type');
     }
   });
-  await store.dispatch('sdkPlugin/initialize');
-  await watchUntilTruthy(() => store.getters['sdkPlugin/sdk']);
 
   connectionsQueue.forEach(addAeppConnection);
   connectionsQueue = [];
 }
 
-export function disconnect() {
-  Object.values(store.getters['sdkPlugin/sdk'].rpcClients).forEach((aepp) => {
+export async function disconnect() {
+  const { getSdk } = useSdk13({ store });
+  const sdk = await getSdk();
+  Object.values(sdk._clients).forEach((aepp) => {
```

Comment:
Properties starting from an underscore in many cases are considered private ones. Maybe the SDK has some kind of getter for this value?
@davidyuk FYI